vhpidirect/quickstart/logicvector/run.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env sh
+
+cd "$(dirname $0)"
+
+set -e
+
+ghdl -a tb.vhd
+ghdl -e -Wl,-I./ -Wl,main.c tb &&
```

Comment:
Why `&&`?

Also, I think that `-Wl,-I./` is not required here.

Reply:
The && makes the execution of the next line dependent on the current line succeeding. Nice for catching compilation bugs cleanly.

Reply:
I think that `set -e` above fulfills the purpose of `&&`.